### DISCUSSION TITLE
How to get started

### DISCUSSION THREAD

**Discussion by brunant** [villekol/tau-latex-thesis-template#3] [Q&A]
Hi,
I'm quite new to Overleaf and github. I copied the main.tex to a new blank project but it won't compile. What should I do with this info to make it work:

pdflatex main.tex
makeindex -s main.ist -t main.glg -o main.gls main.glo
biber main
pdflatex main.tex
pdflatex main.tex

Where shoud I input these lines?

**Comment by AMaybellP358** [comment]
When you create a new project you have to select the option "Upload Project" and upload the .zip of the template. Those lines are only in case you are using the terminal for the compilation. In Overleaf, you will just need to click "Recompile".
